tools/ci_build/github/azure-pipelines/post-merge-jobs.yml

```diff
@@ -446,14 +446,19 @@ stages:
         python tools/ci_build/github/apple/build_apple_framework.py \
           --build_dir "$(Build.BinariesDirectory)/ios_framework" \
           --build_dynamic_framework \
-          tools/ci_build/github/apple/default_full_apple_framework_build_settings.json
+          tools/ci_build/github/apple/default_mobile_ios_framework_build_settings.json
```

Comment:
nit: can we rename the file so it is more obvious that this file is for a test? e.g., test_ios_framework_build_settings.json.
the "mobile" part of the name is also outdated since it refers to the old mobile package that has been removed.